src/Database/Query.php

```diff
@@ -446,10 +402,9 @@ public function andWhere(...$args)
 	/**
 	 * Attaches a group by clause
 	 *
-	 * @param string|null $group
-	 * @return \Kirby\Database\Query
+	 * @return $this
 	 */
-	public function group(string $group = null)
+	public function group(string|null $group = null): static
```

Comment:
`$this->group` type is `mixed` but this method accept as `string|null`. Not consistent.